test/source/tests/setup.ts

```diff
@@ -17,14 +17,26 @@ import { testConstants } from './tooling/consts';
 import { InboxPageRecipe } from './page-recipe/inbox-page-recipe';
 import { PageRecipe } from './page-recipe/abstract-page-recipe';
 import { TestUrls } from '../browser/test-urls';
-import { ControllablePage } from '../browser';
+import { BrowserHandle, ControllablePage } from '../browser';
 import { OauthPageRecipe } from './page-recipe/oauth-page-recipe';
+import { AvaContext } from './tooling';
 
 // tslint:disable:no-blank-lines-func
 // tslint:disable:no-unused-expression
 /* eslint-disable no-unused-expressions */
 /* eslint-disable max-len */
 
+const getAuthorizationHeader = async (t: AvaContext, browser: BrowserHandle, acctEmail: string) => {
```

Comment:
Does this always return `Bearer undefined` for the new 3 tests?
If so, why bother? We could just as well send `Bearer undefined` or something more descriptive like `Bearer emulating-not-properly-registered-extension` right away.

Reply:
If we want to double-check the test setup itself (the fact that the extension isn't actually set up), we can try to extract google access token and `expect` it to be `undefined` right here (or in the beginning of each test). This would be more obvious, then sending incorrectly extracted value.